frontend/mr-admin-flate/src/components/modal/RedigeringsAdvarselModal.tsx

```diff
@@ -0,0 +1,37 @@
+import styles from "./Modal.module.scss";
+import { Heading, BodyShort, Button, Modal } from "@navikt/ds-react";
+import { RefObject } from "react";
+
+interface Props {
+  modalRef: RefObject<HTMLDialogElement>;
+  onRediger: () => void;
+}
+
+export const RedigeringsAdvarselModal = ({ modalRef, onRediger }: Props) => {
+  const onClose = () => {
+    modalRef.current?.close();
+  };
+
+  return (
+    <Modal ref={modalRef} onClose={onClose} aria-label="modal">
+      <Modal.Header closeButton={false}>
+        <div className={styles.heading}>
+          <Heading size="medium">Du er ikke eier av denne ressursen</Heading>
```

Comment:
Er ressursen et ganske teknisk begrep? Burde vi vurdere noe annet?

Hva med `Du er ikke administrator for denne avtalen/gjennomføringen`. Avtalen/gjennomføringen kan evt. sendes inn som en prop for å skille på teksten?

Reply:
Sendte inn med prop 👍